### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 47

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#235] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255539-42fc573c-ce98-4f69-a38e-f95c5dc2794d.png)

**Comment by i0nut02** [comment]
mamaorossi 
1. 10!/ (2! * 2! * 2! * 2!)
2. 10!/ (2! * 2! * 2! * 2!) - 9! / (2! * 2! * 2!)
3. 10!/ (2! * 2! * 2! * 2!) - (   8! / (2! * 2!) + 8! / (2! * 2!) - 6!(2! * 2!)   )

**Comment by Elia-Belli** [comment]
$S = ELIABELLI$
- **1)** Gli anagrammi di una parola sono le disposizioni semplici delle sue lettere: $$\frac {9!}{2!3!2!}$$
- **2)** Gli anagrammi di $S$ che non iniziano con $E$ sono tutti gli anagrammi possibili - quelli che iniziano per $E$ (ovvero gli anagrammi delle restanti lettere togliendo una $E$):
$$(\frac {9!}{2!3!2!}) - (1*\frac {8!}{2!2!2!})$$
- **3)** Gli anagrammi di $S$ che non iniziano né finiscono con $E$ sono tutti gli anagrammi possibili - quelli che iniziano e finiscono per $E$ (ovvero gli anagrammi delle restanti lettere togliendo due $E$):
$$(\frac {9!}{2!3!2!}) - (1*\frac {7!}{2!2!}*1)$$

**Osservazione**: siccome l'esercizio si basa sul nome e cognome della persona che lo risolve non ci sono soluzioni uguali, ma il metodo risolutivo è lo stesso
